run_releasenotes_check.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+set -e
+
+DIR="$(dirname "$0")"
+"$DIR/run.sh" "$DIR/src/run_releasenotes_check.py" $@
```

Comment:
I would remove this script, too many scripts at the root. Maybe just call `run.sh` from the Jenkins job?

Otherwise I think it could be called `release_notes.sh`, and have a "check" command line function, so `release_notes.sh check manifests/manifest.yml`. This way it becomes extensible for future commands related to release notes (I'm thinking `gather` and `generate` as future improvements). WDYT?

Reply:
Added `check` @dblock , can be called now with `./release_notes.sh check manifests/2.2.0/opensearch-2.2.0.yml --date 2022-07-26`.